src/adaptive_exp_and_gram.jl

```diff
@@ -1,72 +1,68 @@
 """
     AdaptiveExpAndGram{T,A} <: AbstractExpAndGramAlgorithm
 
-Adaptive algorithm for computing the matrix exponential and an associated Gramian. 
+Adaptive algorithm for computing the matrix exponential and an associated Gramian.
 
-Constructor: 
+Constructor:
 
 AdaptiveExpAndGram{T}()
 
-creates an algorithm with coefficients stored in the numeric type T. 
+creates an algorithm with coefficients stored in the numeric type T.
 
 """
 struct AdaptiveExpAndGram{T,A} <: AbstractExpAndGramAlgorithm where {A}
     methods::A
+    function AdaptiveExpAndGram{T}() where {T}
```

Comment:
Made the constructor an inner constructor to make sure that you can't ever build this with some other `methods` passed